operation/fhir-operation-bulkdata/pom.xml

```diff
@@ -23,6 +23,21 @@
             <groupId>jakarta.ws.rs</groupId>
             <artifactId>jakarta.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jacorb</groupId>
+                    <artifactId>jacorb-omgapi</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.spec.javax.rmi</groupId>
+                    <artifactId>jboss-rmi-api_1.0_spec</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
can we remove this altogether now?  if not, why not?

Reply:
I'll paste error I was getting when I tried to removed it.

Reply:
```
[ERROR] testCheckAndConvertToMediaType(com.ibm.fhir.operation.bulkdata.util.BulkDataExportUtilTest)  Time elapsed: 0.024 s  <<< FAILURE!
java.lang.RuntimeException: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
	at com.ibm.fhir.operation.bulkdata.util.BulkDataExportUtilTest.testCheckAndConvertToMediaType(BulkDataExportUtilTest.java:152)
Caused by: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
	at com.ibm.fhir.operation.bulkdata.util.BulkDataExportUtilTest.testCheckAndConvertToMediaType(BulkDataExportUtilTest.java:152)
```

Reply:
Yes, we have the client in place.  Note, I recently changed to use the single cxf implementation, otherwise we'd have to include the Jersey Client.

Reply:
So, to confirm, I do need to leave the 'provided' dependency in place?  Or is there a way to remove it altogether?

Reply:
you have to leave it for now.

Reply:
would setting it to test scope work?